src/main/java/org/kohsuke/github/GHAppInstallation.java

```diff
@@ -365,4 +365,9 @@ public GHAppCreateTokenBuilder createToken(Map<String, GHPermissionType> permiss
     public GHAppCreateTokenBuilder createToken() {
         return new GHAppCreateTokenBuilder(root, String.format("/app/installations/%d/access_tokens", getId()));
     }
+
+    GHAppInstallation wrap(GitHub root) {
+        this.root = root;
+        return this;
+    }
```

Comment:
```suggestion
```

I don't think you need this now. `root` gets bound by Jackson automatically. 

https://github.com/hub4j/github-api/blob/81f23e8ae51218b63001c10077a016f07093deae/src/main/java/org/kohsuke/github/GitHubInteractiveObject.java#L13-L14

Reply:
Also if needed there's already a `wrapUp()` on line 313.